src/client/driver/command-executors/prepare-browser-manipulation.js

```diff
@@ -14,7 +14,7 @@ const RESTORE_DOCUMENT_TITLE_TIMEOUT = 100;
 
 export default function prepareBrowserManipulation (testRunId) {
     var savedDocumentTitle   = document.title;
-    var assignedTitle        = `[ ${window.location.toString()} ]`;
+    var assignedTitle        = `[ ${testRunId} ]`;
```

Comment:
Please, fix also line 30 in this file:
`document.title = testRunId;`

I've messed it up in my PR, sorry :disappointed: